### PR TITLE
Language: add hook to sort entries in language files

### PR DESCRIPTION
**Pull request by nhaagen**
Following #3562 and a JF-vote for sorted lang-entries, here is my suggestion on a pre-commit hook.

### CODE REVIEW

**Comment by mjansenDatabay** [comment]
Great @nhaagen !

Does this work on Windows machines? AFAIK @mbecker-databay still uses Windows for development ;-), and if he'd like to use the `Caiptn`, this will probably fail. 

The question is: Do we care about developers using Windows :)?

**Comment by nhaagen** [comment]
Well, I think we do care, since we are nice people and all ;), and I expect Windows to be of more relevance especially when it comes to translations. I'll find somebody to check into this.

**Comment by nhaagen** [comment]
I verified with @catkrahl in both git-bash and power shell: it works.

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
I suggest to move the script to the `CI` folder. I don't have any other objections.

But I don't know if I am the only person who decides upon changes applied to the `captainhook.json`. So maybe we should put this on a JF agenda and discuss this.

**Comment by matthiaskunkel** [comment]
Thanks, @mjansenDatabay, for your review and reply. I have added this PR to the agenda of the upcoming JourFixe at Dec 13, 2021.

**Comment by nhaagen** [comment]
Hey @mjansenDatabay,
I moved the script to CI, langfolder was awkward anyway.
Cheers

File: captainhook.json [hunk 1]
```diff
@@ -48,6 +48,20 @@
             ]
           }
         ]
+      },
+      {
+        "action": "CI/sort_langfile_entries.sh}",
```
Comment (mjansenDatabay) [inline captainhook.json]: The `}` looks like a typo ;-)

Comment (nhaagen) [reply captainhook.json]: thank you, sure is :)

**Comment by matthiaskunkel** [comment]
Jour Fixe, 13 DEC 2021 : We highly appreciate this suggestion and accept the PR for trunk and release 6 and 7.

**Comment by matthiaskunkel** [comment]
Hi @nhaagen, I am bit too late with reviewing this PR. Now it has conflicts. Sorry. Can you solve them? I really would appreciate to introduce the sorting of language files.
Best, matthias

**Comment by nhaagen** [comment]
Hi @matthiaskunkel , 
I resetted to trunk and applied sorting. HIt "merge" quickly :)
Cheers, Nils

**Comment by matthiaskunkel** [comment]
Thanks to @nhaagen for this PR. Accepted and merged to trunk.

**Pull request merged**